doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -222,6 +222,7 @@ nRF Desktop
     This is done to prevent the device from powering down and waking up multiple times when an USB cable is connected.
   * Disabled ``CONFIG_BOOT_SERIAL_IMG_GRP_HASH`` in MCUboot bootloader release configurations of boards that use nRF52820 SoC.
     This is done to reduce the memory consumption.
+  * Motion generation in the Motion Simulated module is now based on the cycle count and uses a timestamp in microseconds instead of a timestamp in milliseconds to improve accuracy.
```

Comment:
Add ref to module's doc page, please. also consider linking to `k_cycle_get_32` and `k_cycle_get_64` APIs to make it clear.